operate/quickstart/run_service.py

```diff
@@ -136,28 +136,47 @@ def configure_local_config(template: ServiceTemplate) -> QuickstartConfig:
     if config.staking_vars is None:
         print_section("Please, select your staking program preference")
         ids = list(template["staking_programs"].keys())
+        available_choices = {}
         for index, key in enumerate(ids):
             metadata = staking_handler.get_staking_contract_metadata(program_id=key)
             name = metadata["name"]
             description = metadata["description"]
             wrapped_description = textwrap.fill(
                 description, width=80, initial_indent="   ", subsequent_indent="   "
             )
-            print(f"{index + 1}) {name}\n{wrapped_description}\n")
-
+            program_id = key
+            if name.lower().startswith('no staking'):
```

Comment:
we have a constant for that
```suggestion
            if program_id == NO_STAKING_PROGRAM_ID:
```

Reply:
updated